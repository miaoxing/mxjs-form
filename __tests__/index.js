import {render} from '@testing-library/react';
import {Form} from '..';
import {MemoryRouter} from 'react-router';

describe('test', () => {
  test('basic', () => {
    const result = render(<MemoryRouter>
      <Form initialValues={{}}/>
    </MemoryRouter>);
    expect(result.container).toMatchSnapshot();
  });
});
