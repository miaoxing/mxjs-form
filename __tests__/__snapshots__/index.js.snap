// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test basic 1`] = `
<div>
  <form />
</div>
`;
